Makefile

```diff
@@ -11,8 +11,13 @@ default : install
 build : test
 	CGO_ENABLED=0 go build -trimpath -ldflags "$(LDFLAGS)" .
 
-install : test
+install : build 
 	CGO_ENABLED=0 go install -trimpath -ldflags "$(LDFLAGS)" github.com/gittuf/gittuf
 
 test :
 	go test -v ./...
+
+.PHONY: fmt
```

Comment:
Can you put this in the .PHONY line at the top of the file with the other targets?

Reply:
Fixed